mu4/domain/notation/inotationstyle.h

```diff
@@ -0,0 +1,22 @@
+#ifndef INOTATIONSTYLE_H
+#define INOTATIONSTYLE_H
+
+#include <QVariant>
+
+#include "modularity/imoduleexport.h"
+#include "libmscore/style.h"
+
+using StyleId = Ms::Sid;
+
+namespace mu::domain::notation {
+class INotationStyle : MODULE_EXPORT_INTERFACE
+{
+public:
+    ~INotationStyle() = default;
+
+    virtual void updateStyleValue(const StyleId& styleId, const QVariant& newValue) const = 0;
+    virtual QVariant styleValue(const StyleId& styleId) = 0;
```

Comment:
Strange, the `updateStyleValue` method is const, and the `styleValue` method is not const, it seems it should be the other way around. Explain why this is so?

Reply:
Oh, I got them mixed up due to inattention, ty